guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -85,23 +85,36 @@ ifdef::katello[]
 ----
 # foreman-rake katello:upgrade_check
 ----
-. Update operating system packages
+. Update operating system packages:
 +
 [options="nowrap" subs="attributes"]
 ----
 # yum update -y
 ----
 . Update repositories
 +
-.For Centos 7 or {RHEL} Users:
+.For Centos 7 or {RHEL} 7 Users:
 [options="nowrap" subs="attributes"]
 ----
-# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
-# yum update -y https://yum.theforeman.org/katello/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm \
+                https://yum.theforeman.org/katello/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
 # yum install -y centos-release-scl-rh
 ----
-. Update packages
-Clean the yum cache and update the required packages:
++
+.For Enterprise Linux 8 or {RHEL} 8 Users:
+[options="nowrap" subs="attributes"]
+----
+# dnf update -y https://yum.theforeman.org/releases/{ProjectVersion}/el8/x86_64/foreman-release.rpm \
+                https://yum.theforeman.org/katello/{KatelloVersion}/katello/el8/x86_64/katello-repos-latest.rpm
+----
+// This step is only needed for upgrades to Katello 4.5 and later
```

Comment:
```suggestion
```

This comment is not necessary because we will only merge to master and cherry-pick to Foreman 3.3/Katello 4.5 as you've selected.

Reply:
Okay thank you, removed the line